week1/homework/exercises/3-web-server/server.js

```diff
@@ -2,13 +2,53 @@
  * Exercise 3: Create an HTTP web server
  */
 
-var http = require('http');
-
-//create a server
-let server = http.createServer(function (req, res) {
-  // YOUR CODE GOES IN HERE
-	res.write('Hello World!'); // Sends a response back to the client
-	res.end(); // Ends the response
+const http = require('http');
+const fs = require('fs');
+const port = 3000;
+const index = 'index.html';
+const style = 'style.css';
+const indexJS = 'index.js';
+const server = http.createServer((req, res) => {
+    if (req.url === '/') {
+        fs.readFile(index, (err, data) => {
+            if (err) {
+                res.writeHead(404);
+                res.write('Error :File not found');
```

Comment:
nice, I would add a log message for the developer
```suggestion
                console.error(err);
                res.writeHead(404);
                res.write('Error :File not found');
```

Reply:
i will add a log , thank you